src/main.tf

```diff
@@ -128,6 +125,4 @@ resource "azurerm_cosmosdb_account" "main" {
       storage_redundancy  = var.backups.redundancy
     }
   }
-
-  tags = var.md_metadata.default_tags
 }
```

Comment:
Disabling local authentication disables the primary/secondary keys that are used in the artifact. This would break the artifact for the bundle. Until we support RBAC auth between Cosmos DB and other bundles, I don't recommend enabling this.

Reply:
This may be something that we either give the user the option, or hardcode to username/password.  I think the IAM based auth in all the clouds requires using the Cloud's SDK libraries to generate the auth token.  This complicates things like local development and application portability.

Reply:
Well right now we pass the primary key as the password in the artifact. Disabling local authentication would remove the keys to auth. We don't have another way to connect other services to the cosmos DB if the keys are disabled.